p2panda-js/jest.config.js

```diff
@@ -0,0 +1,19 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
```

Comment:
I am asking myself whether it would be possible to test p2panda-js in a browser environment using jsdom in the future.

Reply:
Checking the wasm tests against the browser can also be interesting: https://rustwasm.github.io/wasm-pack/book/tutorials/npm-browser-packages/testing-your-project.html. It does not test the p2panda-js library but at least the hairy wasm part.

```bash
wasm-pack test --headless --firefox
```

Reply:
I like that